erp/import_datasets/base_mapper.py

```diff
@@ -0,0 +1,24 @@
+import abc
+from typing import List, Any
+
+from erp.models import Erp, Activite
+
+
+class BaseRecordMapper(abc.ABC):
+    @property
+    @abc.abstractmethod
+    def dataset_url(self) -> str:
+        pass
+
+    @property
+    @abc.abstractmethod
+    def activite(self) -> str:
+        pass
+
+    @abc.abstractmethod
+    def fetch_data(self) -> List[Any]:
```

Comment:
I'm puzzled with the use of `Any` here; shouldn't we return a more specific type instead?

Reply:
Done